release.py

```diff
@@ -2,22 +2,25 @@
 
 from __future__ import print_function
 from argparse import RawTextHelpFormatter
+from configparser import ConfigParser
 from typing import Tuple
+from urllib3.exceptions import RequestError
+from urllib3.util import make_headers
 import subprocess
 import sys
 import tempfile
 import os
 import urllib.parse
-import urllib.request
+import urllib3
```

Comment:
it would be helpful to document somewhere the python dependencies needed to run `release.py`. I am using a python venv, so I with trial and error I installed the missing dependencies, but it could be clearer

Reply:
Or avoid using libraries outside of the python standard libraries if possible.

Reply:
> it would be helpful to document somewhere the python dependencies needed to run `release.py`. I am using a python venv, so with trial and error I installed the missing dependencies, but it could be clearer

I added basic python metadata in [ccf062b](https://github.com/gazebo-tooling/release-tools/pull/1201/commits/ccf062bf1ffce5cf5186dd067d5948624453614c) and updated the documentation in https://github.com/gazebosim/docs/pull/538

> Or avoid using libraries outside of the python standard libraries if possible.

Need to double check if we can get rid of urlib3 now that I have clear how the credentials work through GET/POST. Don't want to make more changes in this PR, let's try in a different one.